Resources/Textures/Structures/Wallmounts/textscreen.rsi/meta.json

```diff
@@ -0,0 +1,13 @@
+{
+  "version": 1,
+  "size": {
+    "x": 32,
+    "y": 32
+  },
+  "copyright": "brainfood",
```

Comment:
Needs a license, also should really be their entire discord handle and specify as such.